gapic/src/main/com/google/gapid/widgets/LoadableImage.java

```diff
@@ -43,31 +44,37 @@
   protected static final Logger LOG = Logger.getLogger(LoadableImage.class.getName());
 
   private final ListenerCollection<Listener> listeners = Events.listeners(Listener.class);
+  private final AtomicInteger loadCount = new AtomicInteger(0);
```

Comment:
No need for atomic integer. The load and unload methods are assumed to be only called from the UI thread.